app/code/Magento/Catalog/Model/Category.php

```diff
@@ -105,11 +105,6 @@ class Category extends \Magento\Catalog\Model\AbstractModel implements
      */
     protected $_urlRewrite;
 
-    /**
-     * @var ResourceModel\Category
-     */
-    protected $_resource;
-
```

Comment:
while this is not necessary and not relevant for this PR anymore, it helps with static analysis and IDE support by specifying the concrete type of `$_resource`